database/migrations/2022_11_30_145724_update_action_events_table.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class () extends Migration {
+    public function up(): void
+    {
+        Schema::table('action_events', function (Blueprint $table) {
+            $table->string('user_id', 42)->change();
```

Comment:
It's safer to make it `255` while you're at it, because we do not have control over the length of the Auth0 ids. For example when someone logs in via Google or Facebook their id is `google-oauth2|<random string>` or `facebook|<random string>`. We cannot expect the length to be the same for all those social logins forever, and more types of logins can be added in the future (for example Apple and Mijn Burgerprofiel were added recently)